src/pages/index.js

```diff
@@ -1,38 +1,86 @@
 import "./index.css";
 
 //import all the classes
-import { initialCards, selectors, config } from "../utils/constants";
+import { selectors, config } from "../utils/constants";
 import Card from "../components/Card";
 import FormValidator from "../components/FormValidator";
 import Section from "../components/Section";
 import PopupWithImage from "../components/PopupWithImage";
 import PopupWithForm from "../components/PopupWithForm";
 import UserInfo from "../components/UserInfo";
-
+import Api from "../components/Api";
+import PopupWithConfirm from "../components/PopupWithConfirm";
 //Create instances of the classes
+
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "d78649ed-fd14-41f7-9a2b-04c3fb13cc28",
+    "content-type": "application/json",
+  },
+});
+
+const cardSection = new Section(
+  {
+    renderer: (data) => {
+      cardSection.addItems(createCard(data));
+    },
+  },
+  selectors.cardSection
+);
+
 const createCard = (data) => {
-  const card = new Card(data, "#card-template", () => {
-    cardPreviewPopup.open(data);
-  });
+  const card = new Card(
+    data,
+    "#card-template",
+    () => {
+      cardPreviewPopup.open(data);
+    },
+    function handleCardDelete(card) {
+      confirmDeletePopup.open();
+      confirmDeletePopup.setSubmitAction(() => {
+        confirmDeletePopup.setLoading(true, "Deleting");
+        api
+          .removeCard(card.getId())
+          .then(() => {
+            card._handleTrashIcon();
```

Comment:
`_handleTrashIcon`

Please, delete the **low dash** (`underscore`) from methods/fields if you want to use them **outside** a class because the `low dash` shows a **private** field which is used only inside the class.


And correct it in `Card` as well